egs/wsj/s5/utils/lang/check_g_properties.pl

```diff
@@ -73,8 +73,8 @@
   }
 }
 
-close(O);  # tell 'fstcompile | fstinfo' pipeline that its input is done.
-while (<I>) {
+close(I);  # tell 'fstcompile | fstinfo' pipeline that its input is done.
+while (<O>) {
   if (m/cyclic\s+/) {
```

Comment:
Remi, does the check pass for you now?
y.

On Thu, May 5, 2016 at 5:56 AM, Rémi Francis <notifications@github.com>
wrote:

> In egs/wsj/s5/utils/lang/check_g_properties.pl
> <https://github.com/kaldi-asr/kaldi/pull/756#discussion_r62167747>:
>
> > @@ -73,8 +73,8 @@
> >    }
> >  }
> >
> > -close(O);  # tell 'fstcompile | fstinfo' pipeline that its input is done.
> > -while (<I>) {
> > +close(I);  # tell 'fstcompile | fstinfo' pipeline that its input is done.
> > +while (<O>) {
> >    if (m/cyclic\s+/) {
>
> Right, that is obvious in hindsight...
> I've updated the pull request.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/756/files/a8b32fd28cc411160741475a6e0b969fb5d9fc37#r62167747>
>